mac.sh

```diff
@@ -65,7 +65,7 @@ brew_install_or_upgrade() {
 brew_is_installed() {
   local name="$(brew_expand_alias "$1")"
 
-  brew list -1 | grep -Fqx "$name"
+  brew list --formula -1 | grep -Fqx "$name"
```

Comment:
In case you're asking "but is `brew_is_installed` also checking for casks?", this function `brew_is_installed` is specifically to check if a formula is installed and is called by `brew_install_or_upgrade`. We also have parallel functions `cask_is_installed` and `cask_install_or_upgrade`.